src/screens/JobRun/JobRunView.test.tsx

```diff
@@ -54,4 +55,46 @@ describe('JobView', () => {
     // Task Runs Card
     expect(queryByText(/jsonparse/i)).toBeInTheDocument()
   })
+
+  it('renders the job run view with no errors ', async () => {
+    const run = buildRun({
+      status: 'COMPLETED',
+      allErrors: [],
+      job: {
+        id: '10',
+        name: 'job 1',
+        observationSource: `ds1 [type=bridge name="bridge-api0"] ds2 [type=bridge name="bridge-api1"]`,
+      },
+      taskRuns: [
+        buildTaskRun({ dotID: 'ds1' }),
+        buildTaskRun({ dotID: 'ds2' }),
+      ],
+    })
+
+    renderComponent(run)
+
+    // Heading
+    expect(queryByText(`Job Run #${run.id}`)).toBeInTheDocument()
+
+    // Job Run Card
+    expect(queryByText(run.id)).toBeInTheDocument()
+
+    // Tabs
+    expect(queryByRole('tab', { name: 'Overview' })).toBeInTheDocument()
+    expect(queryByRole('tab', { name: 'JSON' })).toBeInTheDocument()
+
+    // Task List Card
+    expect(queryByText(/task list/i)).toBeInTheDocument()
+
+    // ds1 and ds2 are correctly displayed
+    const ds1 = queryByText(/ds1/, { selector: 'text' })?.parentNode?.firstChild
+    const ds2 = queryByText(/ds2/, { selector: 'text' })?.parentNode?.firstChild
+
+    // eslint-disable-next-line @typescript-eslint/ban-ts-ignore
+    // @ts-ignore because material UI doesn't update theme types with options
+    expect(ds1).toHaveAttribute('fill', theme.palette.success.main)
+    // eslint-disable-next-line @typescript-eslint/ban-ts-ignore
+    // @ts-ignore because material UI doesn't update theme types with options
```

Comment:
Better to use `ts-expect-error`, it's more strict

Reply:
Fixed in [`cf2931d` (#29)](https://github.com/smartcontractkit/operator-ui/pull/29/commits/cf2931ddc3304a2bbb14d93706830827a52f159e)